gef-ui/server/utils/helpers.test.js

```diff
@@ -559,8 +566,12 @@ describe('mapSummaryList()', () => {
     };
 
     mockedDisplayItems[0].id = 'coverStartDate';
-    mockedDisplayItems[0].method = (callback) => moment(callback)
-      .format('D MMMM YYYY');
+    mockedDisplayItems[0].method = (value) => {
+      // value is an ISO-8601 string with milliseconds (e.g '2024-02-14T00:00:00.000+00:00')
```

Comment:
This whole test file could be rationalised a bit and made DRYer by extracting common setup -- but not in this review or ticket as it could be large task. Good identification of scope to be honest, as I would have been tempted.

```suggestion
      // input value is an ISO-8601 string with milliseconds (e.g '2024-02-14T00:00:00.000+00:00')
```